src/layer6.twee

```diff
@@ -761,6 +761,40 @@ You have successfully taken on the $curse77.name curse. Hopefully you are ok wit
 	[[Return to your balloon|Escape Balloon L6]]
 <</if>>
 
+:: Horny Customization
+What type of horn(s) would you like to gain?
+
+<<radiobutton "$curseData[77].variation" "demonic" checked>> Demon
+<<radiobutton "$curseData[77].variation" "draconic">> Dragon
+<<radiobutton "$curseData[77].variation" "unicorn">> Unicorn
+<<radiobutton "$curseData[77].variation" "rhino">> Rhino
+
+How many horns would you like to gain?
+
+<<radiobutton "$curseData[77].count" 1 checked>> One <<radiobutton "$curseData[77].count" 2 >> Two
```

Comment:
It might make more sense to have the number of horns be implied by the type. For example, ["unicorn" literally means "one horn"](https://www.etymonline.com/word/unicorn).

Is demon horn a single type? I've seen demon horns drawn as smooth cow-like horns, as ribbed goat-like horns and as large curved sheep-like horns.

Reply:
I started off with only 1 unicorn horn but then FloricSpacer said "I mean, I don’t see why someone couldn’t get 2 unicorn style horns if they wanted?"

I could add more demon horn types although this could be done with the adjectives.